README.md

```diff
@@ -248,7 +248,7 @@ Retrieves the current state of a payment rail.
 
 #### `terminateRail(uint256 railId)`
 
-Emergency termination of a payment rail, preventing new payments after the lockup period. This should only be used in exceptional cases where the operator contract is malfunctioning and refusing to cancel deals.
+Normal termination of payment that enters the rail into its final lockup period before finalizing completion of the payment flow. This can be called by the operator or a client in good standing.
```

Comment:
This sentence seemed to be a bit of a mouthful, at least the bolded out parts of it:
>Normal termination of payment that enters the rail **into its final lockup period before finalizing completion of the payment flow.**

But this might also just be that English is not my native language.

Reply:
All good docs should be easy for anyone who is ESL so this is good feedback.  I'll make it better

Reply:
No it reads better for me at least 🙏